tests/foreman/cli/test_contentview.py

```diff
@@ -5323,7 +5322,26 @@ def test_positive_arbitrary_file_repo_promotion(self):
         :expectedresults: Check arbitrary files from FR is available on
             environment
 
-        :CaseAutomation: notautomated
+        :CaseAutomation: automated
 
         :CaseLevel: Integration
         """
+        org = ContentViewFileRepoTestCase.org
```

Comment:
why not self.org?  Just to make code consistent

Reply:
Agreed, Omkar same has been taken care.